pattern_generator.rb

```diff
@@ -0,0 +1,79 @@
+class PatternGenerator
+  
+  PATTERNELEMENTS = {"."=>["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M", "N", "O", "P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z"], "#"=>['0', '1', '2', '3', '4', '5', '6', '7', '8', '9']}
```

Comment:
Dynamic range. So if the code base needs `0..90` you can just change one thing and you still have a valid array of string nums